spec/features/tasks/index_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+feature 'Viewing all tasks' do
+    scenario 'when visiting the page the first time, View All link is available' do
+        visit tasks_path
+        expect(page).to have_content("View All")
```

Comment:
I wonder, is there another capybara helper you could use here that would ensure the content "View All" is the link you are looking for with the correct href?

Reply:
```ruby
visit tasks_path
        
expect(page).to have_link("View All", href: tasks_path(all: true))
```

Reply:
Nice!